wiki/Beatmapping/en.md

```diff
@@ -1,337 +1,56 @@
+### Song Setup
 
-It is also not uncommon for [mappers](Mappers "wikilink") to use custom [difficulty](Difficulty "wikilink") names in place of these. (Like "Mario" for example)
+_See also: [Song Setup](/wiki/Song_Setup)._
 
-Additionally you may find [Guest Difficulties](Guest_Difficulty "wikilink") which are commonly labeled as mapper's difficulty (For example: "Larto's Insane").
+When making a new [mapset](/wiki/mapset), you will always see this dialog before any other parts of the beatmap editor.
 
-With the addition of the Special modes, you can now add mod-specific difficulties to your beatmap other than the usual osu! difficulty. It is entirely possible to fill the beatmap with only Taiko/Catch the Beat/Osu!mania difficulties provided there is an Easy/Normal difficulty in it.
+You should fill out this information before you create another [difficulty](/wiki/difficulty).
+This will ensure that the metadata will be consistent across all of the difficulties in the set.
 
-Getting Started
-===============
+You can change the options in the other tabs now if you want, but they can be changed later on.
+Click `OK`, and you can start beatmapping!
+The next step is to add timing to your beatmap.
 
-How do I start a new beatmap?
------------------------------
+### Timing
 
-As osu! is a rhythm based game, the songs you choose to map should have an obvious and constant beat. Try to avoid songs which have tempo (speed) changes or slow songs until you are more experienced and used to the beatmapping mechanism.
+_See also: [Timing](/wiki/Timing)._
 
-To start a new beatmap, follow these simple steps:-
+**Timing is vital!**
+An incorrectly timed maps will not be considered for ranking.
+You must make sure that your timing is perfect before adding any [hit objects](/wiki/hit_objects).
 
--   Drag and drop the audio file onto osu! (Opened or not is not an issue here)
+If you are in doubt about timing, contact a mod for help first, or submit your song into the [Beatmap Help forum](https://osu.ppy.sh/forum/10).
+This will save you (and the modding community) a lot of hassle later on.
 
-Alternative route: Put audio file onto the "Songs" folder.
+### Beatmap Editor
 
--   It should open and direct you to the edit screen with your audio file at there.
--   Your new song will be at the bottom of the song list, highlighted in purple.
+_See also: [Beatmap Editor](/wiki/Beatmap_Editor)._
 
--   Click on the song to open up Song Setup.
-    -   Insert and check that the song title and artist are correct, and add Source and Tags if necessary, then enter in an appropriate difficulty name (Easy, Insane and Cruisin' are examples, you can change it later anyway).
--   You can set the other options now if you want, but they can be changed later on. Click OK, and you can start beatmapping! The next step is to add timing to your beatmap.
+The beatmap editor is where you will probably spend, if not, a majority of your time here in the beatmap editor.
```

Comment:
Rephrase I guess.
`The beatmap editor is where you will probably spend most, if not alll of your time beatmapping.`